Sources/Request Strategies/MLS/Actions/SendMLSMessagesAction.swift

```diff
@@ -0,0 +1,90 @@
+//
+// Wire
+// Copyright (C) 2022 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+public final class SendMLSMessagesAction: EntityAction {
+
+    // MARK: - Types
+
+    public typealias Result = Void
+
+    public enum Failure: LocalizedError {
+
+        case invalidBody
+        case mlsProtocolError
+        case missingLegalHoldConsent
+        case legalHoldNotEnabled
+        case mlsProposalNotFound
+        case mlsKeyPackageRefNotFound
+        case noConversation
+        case mlsStaleMessage
+        case mlsClientMismatch
+        case mlsUnsupportedProposal
+        case mlsUnsupportedMessage
+        case endpointNotAvailable
```

Comment:
```suggestion
        case endpointUnavailable
```

for consistency with the other actions